commons/src/main/java/org/infinispan/commons/dataconversion/BinaryEncoder.java

```diff
@@ -53,7 +53,7 @@ private static boolean isTypeExcluded(Class<?> type) {
    }
 
    private boolean skipEncoding(Object source) {
-      return isTypeExcluded(source.getClass());
+      return source == null || isTypeExcluded(source.getClass());
```

Comment:
This is something lost in translation from the TypeConverters, where ```null``` was never 
boxed or unboxed, but returned "as is". 

Without this change, the null was being passed down to the GlobalMarshaller and it was being serialized as a ```byte```. Not sure why it was breaking @karesti commands, possibly the marshaller was not unmarshalling it back as ```null```?